test_conformance/images/kernel_read_write/test_common.cpp

```diff
@@ -161,6 +249,15 @@ int test_read_image(cl_context context, cl_command_queue queue,
     size_t threads[3];
     static int initHalf = 0;
 
+    size_t image_size =
+        get_image_num_pixels(imageInfo, imageInfo->width, imageInfo->height,
+                             imageInfo->depth, imageInfo->arraySize);
+    test_assert_error(0 != image_size, "Invalid image size");
+    size_t width_size, height_size, depth_size;
+    test_error(
```

Comment:
If _get_image_dimensions_ returns a _test_status_, then that's not a CL error code, and so _test_error_ isn't the right thing to be using here, either.

Reply:
test_read_image always returns an error code, so I have return one here.

Reply:
Seems reasonable, at least for now.